api/README.md

```diff
@@ -21,14 +21,56 @@ __Parameters__
 
 __`sha`__ : SHA[0:10] of the runs to get, or the keyword `latest`. Defaults to `latest`.
 
+__`product`__ : Product to include (repeated param), e.g. `chrome` or `firefox-60`
+
 __`from`__ : RFC3339 timestamp, for which to include runs that occured after the given time.
 
 __`max-count`__ : Maximum number of runs to get (for each browser). Only relevant when `sha` is `latest`. Maximum of 500.
```

Comment:
What's the relevance of `max-count`, and why does it only have an affect for `latest`?

Reply:
If you select a specific SHA, you'll (probably) be getting a single run... techincally, though, there can be multiple runs for the same SHA so this comment is not strictly true. I'll remove it.